adapters/adform/adform.go

```diff
@@ -466,7 +466,7 @@ func openRtbToAdformRequest(request *openrtb.BidRequest) (*adformRequest, []erro
 
 	eids := ""
 	consent := ""
-	if request.User != nil {
+	if request.User != nil && request.User.Ext != nil {
```

Comment:
Not strictly required, but adding as a slight performance boost. No reason to try unmarshaling when we know it will fail.